packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -37,9 +37,9 @@ const LOG_TAG = 'Connection';
 // TODO(b/38203344): The SDK_VERSION is set independently from Firebase because
 // we are doing out-of-band releases. Once we release as part of Firebase, we
 // should use the Firebase version instead.
-const X_GOOG_API_CLIENT_VALUE = `gl-node/${
-  process.versions.node
-} fire/${SDK_VERSION} grpc/${grpcVersion}`;
+const X_GOOG_API_CLIENT_VALUE = `gl-node/${process.versions.node} fire/${
+  SDK_VERSION
+} grpc/${grpcVersion}`;
```

Comment:
You probably need to run "yarn" to get the new version of prettier so these diffs go away.

Reply:
@mikelehen Do I need to manually revert the prettier commit and push again after running yarn?

Reply:
I _think_ just running `yarn` and then doing a `git push` should revert the changes, since the pre-push hook will run prettier again and it should undo the changes.

Reply:
Looks like the pre-pusher reverted the previous change in the new commit. Thank you so much @mikelehen

Reply:
Thanks for this @mikelehen